cfgrib/bindings.py

```diff
@@ -53,12 +53,12 @@ def __getattr__(self, attr):
 for libname in LIBNAMES:
     try:
         lib = ffi.dlopen(libname)
-        LOG.info("ecCodes library found using name '%s'.", libname)
+        LOG.debug("ecCodes library found using name '%s'.", libname)
         break
     except OSError as exc:
         # lazy exception
         lib = RaiseOnAttributeAccess(exc, 'ecCodes library not found on the system.')
-        LOG.info("ecCodes library not found using name '%s'.", libname)
+        LOG.debug("ecCodes library not found using name '%s'.", libname)
```

Comment:
@b8raoult this one should be a warning I think.

Also would you mind changing the error message to report all the `LIBNAMES` we have tried instead of just `libname`. Thanks.